Modules/+Drivers/ArduinoServo.m

```diff
@@ -0,0 +1,64 @@
+classdef ArduinoServo < Modules.Driver
+    %ARDUINOSERVO Connects with a hwserver Arduino and uses one pin to control a servo.
+    %
+    % Call with the 1) IP of the host computer (singleton based on ip), and 2) the (integer) pin.
+    
+    properties (Constant)
+        hwname = 'Arduino';
+    end
+    properties (SetAccess=immutable)
+        connection
+        pin
+    end
+    properties (GetObservable, SetObservable)
+        angle = Prefs.Double(NaN, 'min', 0, 'max', 180, 'set', 'set_angle', 'allow_nan', true);
+    end
+    methods(Static)
+        function obj = instance(ip, pin)
+            mlock;
+            pin = round(pin);
+            persistent Objects
+            if isempty(Objects)
+                Objects = Drivers.ArduinoServo.empty(1,0);
+            end
+            [~,resolvedIP] = resolvehost(ip);
+            for i = 1:length(Objects)
+                if isvalid(Objects(i)) && isequal(resolvedIP, {Objects(i).singleton_id, pin})
```

Comment:
I don't believe this is the logic you want; you're `singleton_id` already contains `{resolvedIP, pin}` - perhaps:
`isequal({resolvedIP, pin}, Objects(i).singleton_id)`